docs/development/extensions-contrib/prometheus.md

```diff
@@ -43,7 +43,7 @@ All the configuration parameters for the Prometheus emitter are under `druid.emi
 |`druid.emitter.prometheus.addHostAsLabel`|Flag to include the hostname as a prometheus label.|no|false|
 |`druid.emitter.prometheus.addServiceAsLabel`|Flag to include the druid service name (e.g. `druid/broker`, `druid/coordinator`, etc.) as a prometheus label.|no|false|
 |`druid.emitter.prometheus.pushGatewayAddress`|Pushgateway address. Required if using Pushgateway strategy|no|none|
-
+|`druid.emitter.prometheus.flushPeriod`|Emit metrics to Pushgateway every `flushPeriod` seconds. Required if `pushgateway` strategy is used.|no|15|
```

Comment:
Maybe use `flushMillis` instead as http emitter seems to be using, so that there is no ambiguity of the unit.

Reply:
Also, the default value used by HttpEmitter is 60 seconds.

Reply:
`flushPeriod` is what I suggest because it's the same property name used in graphite emitter. 
I think maybe we can unify some configurations and the scheduler among all emitters in someday. @kfaraz